packages/contracts-bedrock/scripts/bundles/ExecuteBundle.s.sol

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// Forge
+import { Script } from "forge-std/Script.sol";
+import { console2 as console } from "forge-std/console2.sol";
+
+// Testing
+import { FFIInterface } from "test/setup/FFIInterface.sol";
+
+// Contracts
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { BundleExecutorFactory, BundleTransaction } from "scripts/bundles/BundleExecutor.sol";
+
+// Interfaces
+import { IProxyAdmin } from "src/universal/interfaces/IProxyAdmin.sol";
+
+struct SimulationStorageOverride {
+    bytes32 key;
+    bytes32 value;
+}
+
+struct SimulationStateOverride {
+    address contractAddress;
+    SimulationStorageOverride[] overrides;
+}
+
+contract ExecuteBundle is Script {
+    /// @notice Executes a bundle of transactions from a Safe JSON file.
+    /// @param _bundleJsonPath Path to the JSON file containing the bundle of transactions.
+    /// @param _proxyAdmin Address of the ProxyAdmin to transfer ownership to (if zero, does nothing).
+    function run(string memory _bundleJsonPath, IProxyAdmin _proxyAdmin) public {
+        // Etch the FFIInterface.
+        FFIInterface ffi = FFIInterface(address(uint160(uint256(keccak256(abi.encode("optimism.ffi"))))));
```

Comment:
This could be nice to turn into a small, reusable contract that does this logic in `setUp`, making it portable for tests that need it. For example the hashing/encoding tests need it and inherit from `CommonTest` when they dont need to, would be a nice tech debt fix